src/components/Login/Login.scss

```diff
@@ -0,0 +1,55 @@
+@import "./../../styles/variables";
+@import "./../../styles/constants";
+
+.login-form {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-top: 10rem;
+
+  &__title {
+    color: $link-color;
+    margin-bottom: 2rem;
+  }
+
+  &__form {
+    width: 20%;
```

Comment:
what if we remove this?

Reply:
If we remove this, it just appears by default in the form size. I think the default size looks small.